package.json

```diff
@@ -65,22 +69,24 @@
     "@types/lodash": "4.17.4",
     "@types/mapbox__mapbox-gl-draw": "^1.4.6",
     "ajv": "8.12.0",
+    "babel-plugin-react-compiler": "0.0.0-experimental-592953e-20240517",
     "blitz": "2.0.9",
     "dompurify": "^3.1.4",
+    "eslint-plugin-react-compiler": "0.0.0-experimental-c8b3f72-20240517",
     "immer": "10.1.1",
     "js-cookie": "3.0.5",
     "jsts": "2.11.3",
     "jsurl2": "2.2.0",
     "lodash": "4.17.21",
     "maplibre-gl": "4.3.2",
-    "next": "14.2.3",
+    "next": "14.3.0-canary.80",
```

Comment:
next rc (15)

Reply:
I did `npm install next@canary babel-plugin-react-compiler` and that is wha came of that, so I think that is fine.